src/main/java/org/example/bit-manipulation/OddEvenUsingBitManipulation.java

```diff
@@ -0,0 +1,22 @@
+package org.example.bit-manipulation;
```

Comment:
Package name should not contain hypen(-). It should be only _bitmanipulation_ (all lower case)